### DISCUSSION TITLE
Github - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#77] [Announcements]
# Github - CS自学指南

CS自学指南

https://csdiy.wiki/%E5%BF%85%E5%AD%A6%E5%B7%A5%E5%85%B7/Github/

**Comment by Windy147** [comment]
未来的世界或许是开源的世界！

**Reply by Lila-Wang** [reply]
thanks！

**Reply by Cpt11** [reply]
[推荐一个机场，可以试用，不贵](https://bafang.vip/#/register?code=0mobpNZJ)

**Comment by Lila-Wang** [comment]
THANKS!!!!

**Comment by 2661603207** [comment]
开源即共产了属于是

**Comment by sixsun666666** [comment]
共产主义

**Comment by fcjz** [comment]
补充一些我个人的建议：

如果没有科学上网的条件又想顺畅的访问GitHub，可以使用fastGitHub等GitHub加速访问工具，亲测很丝滑，这类工具基本上都是开源免费的。

同为技术社区的StackOverflow没有被墙，但是刚开始访问的时候点击接受cookie或者注册登录按钮是不起作用的，因为它使用的是Google的CDN，而Google在国内被墙了，如果有科学上网工具可以忽视这条，如果没有，去浏览器扩展里搜索将GoogleCDN转换成国内CDN的插件安装，应该就好了，如果一段时间突然失灵，把插件禁用一下等一段时间再启用基本上就没事了。

初入GitHub想要找点有趣的东西可以去awesome看看。

Google可以替换为ecosia、yandex或者bing，不要用百度，百度一般搜不出什么有用的东西，建议学好英文，搜索的时候最好使用英文，英文的准确度要高一些。

如果你英文水平不高并使用国内搜索，或者你在看到这条建议之前一直这么做，那么，大部分问题参照博客园的建议，csdn解决一些小问题也是可以的……不过要警惕，这个网站现在水平不高，并且访问起来有点恶心（划到一半弹出登录，不登录无法复制等，据说下载的积分机制也很迷惑，不过我没有真正注册过，只是以访客身份遇到上述问题，不能确定这条的真实性，其实这些基本都是可以使用插件或脚本解决的，不过为什么要那么麻烦地去访问一个csdn呢，直接跳过不好吗）。

以上，支持这本书。

**Reply by duanchang425** [reply]
如果你英语水平不够高，建议你先看看国内教程

**Reply by SleetZhang** [reply]
感谢分享 2024 .6.13亲测有效  访问GitHub很丝滑

**Reply by Andrew-moon** [reply]
也可以使用watt toolkit（原steam++），可以加速GitHub，可以在微软商店下载，当然GitHub里也可以下载(doge)

**Comment by Joye-Yu** [comment]
**感谢飞猪**呜呜

**Comment by Tangzishun-Li** [comment]
计算机组成原理 是可以对应哪门课啊 help

**Reply by juanniaoxx** [reply]
体系结构，但并不完全对应 CS61c是比较好的建议，特别喜欢硬件也可以选择ETH: Computer Architecture,wiki内均有收集。

**Reply by juanniaoxx** [reply]
sorry 应该是ETH Zurich：Digital Design and Computer Architecture这门课

**Comment by Tianbawen** [comment]
赤旗一定会插遍世界！

**Comment by Teamanmade** [comment]
作个记号给以后的自己发现

**Comment by zzzhhhaaannnggg6** [comment]
太感谢了

**Comment by lil-Journey** [comment]
自建日本节点服务器 月租5元不限流量 本人也在用 失效时可联系人工修复 qq:1165116287

**Comment by wenqi42** [comment]
🪜推荐大伙用cloudeflare wrap比较好😊
